test/test_transforms.py

```diff
@@ -1994,6 +1994,17 @@ def test_normalize_3d_tensor():
     torch.testing.assert_close(target, result2)
 
 
+def test_normalize_with_single_value_mean_std():
+    torch.manual_seed(42)
+    n_channels = 3
+    img_size = 10
+    mean = 0.5
+    std = 0.5
+    img = torch.rand(n_channels, img_size, img_size)
+    # Check that this work
+    target = F.normalize(img, mean, std)
```

Comment:
Nit: Since we don't re-use these variable names, we don't really need to declare them. We can just shorten it to

```py
    img = torch.rand(3, 10, 10)
    target = F.normalize(img, mean=0.5, std=0.5)
```

But in reality, we don't actually need this test at all, as it wouldn't give us any error anyway, because the tests aren't checked by mypy. What we should try to do is to use `mean=some_float` or mean=`std` somewhere in our code instead. If there are no occurrence of such a thing, maybe we can just verify manually.

Note that we have a CI job for type annotations as well, called `type_check_python` https://app.circleci.com/pipelines/github/pytorch/vision/15467/workflows/20203b6e-c9ca-47a3-9d77-22148ff68086/jobs/1249720